src/components/app.js

```diff
@@ -24,12 +27,20 @@ class App extends Component {
     });
   };
 
+  handleSearch = (e) => {
+    this.setState({search: e.target.value});
```

Comment:
Is it working properly in the current version? Early `e` is nullified by React due to performance reason and since setState is async function we can not access `e` directly in it.
We had to do it like this:
`
const { value } = e.target;
setState..
`

Reply:
Yep, it's working properly